src/parser/parser.cpp

```diff
@@ -166,7 +161,7 @@ api::Sort Parser::getSort(const std::string& name)
 {
   checkDeclaration(name, CHECK_DECLARED, SYM_SORT);
   assert(isDeclared(name, SYM_SORT));
-  api::Sort t = api::Sort(d_symtab->lookupType(name));
+  api::Sort t = api::Sort(d_solver, d_symtab->lookupType(name));
```

Comment:
Just a minor note on these kinds of assignments, we can actually just write `api::Sort t(d_solver, d_symtab->lookupType(name));` here. Probably out of scope for this PR but just something I noticed.